src/emulator/module/CMakeLists.txt

```diff
@@ -10,5 +10,5 @@ add_library(
 )
 
 target_include_directories(module PUBLIC include)
-target_link_libraries(module PUBLIC host microprofile)
+target_link_libraries(module PUBLIC host microprofile util spdlog)
```

Comment:
This can probably be reverted if `module` doesn't need logging stuff in a public header.